src/main/java/com/easypost/model/Tracker.java

```diff
@@ -148,7 +149,16 @@ public static boolean createList(Map<String, Object> params) throws EasyPostExce
 	public static boolean createList(Map<String, Object> params, String apiKey) throws EasyPostException {
 		String createListUrl = String.format("%s/create_list", classURL(Tracker.class));
 
-		request(RequestMethod.POST, createListUrl, params, Object.class, apiKey);
+		int count = 0;
+		Map<String, Object> newParams = new HashMap<String, Object>();
+		Map<String, Object> trackers = new HashMap<String, Object>();
+		for (Object tracker : (ArrayList)params.get("trackers")) {
```

Comment:
Add a space after casting (ArrayList).
If a user inputs value that is not a list for the tracker, the casting would fail. Should we add a try-catch block to handle this?

Reply:
Sounds good.

Reply:
Will address formatting in a separate branch.